Library/Formula/python-markdown.rb

```diff
@@ -0,0 +1,18 @@
+class PythonMarkdown < Formula
+  desc "Python implementation of Markdown"
+  homepage "https://pypi.python.org/pypi/Markdown"
+  url "https://pypi.python.org/packages/source/M/Markdown/Markdown-2.6.5.tar.gz"
+  sha256 "8d94cf6273606f76753fcb1324623792b3738c7612c2b180c85cc5e88642e560"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", lib/"python2.7/site-packages"
+    system "python", "setup.py", "install", "--prefix=#{prefix}"
```

Comment:
Thanks @DomT4, I updated the Formula to use _*Language::Python.setup_install_args()_.